code.gs

```diff
@@ -1216,10 +1216,29 @@ function getEventsOnDate (eventDate, calendarId) {
  * of his/her contacts scheduled for the next days.
  */
 function main (forceDate) {
+  log.add('main() running.', 'info');
+  
```

Comment:
[Javascript Semistandard style](https://github.com/GioBonvi/GoogleContactsEventsNotifier/blob/master/.github/CONTRIBUTING.md#coding-guidelines) mandates that no line should end with a space, so blank lines like this one should contain no spaces at all (the same applies to line R1222, R1228, R1414, R1415, R1416.